### ISSUE TITLE
Chatbot-ui: Error: connection refused

### ISSUE THREAD

**Issue by DenuxPlays** [go-skynet/LocalAI#496]
**LocalAI version:**
Commit: [203fd7b](https://github.com/go-skynet/LocalAI/commit/203fd7b2e8731d82031f02d1f443b352cd8c5d25)

**Environment, CPU architecture, OS, and Version:**
Linux winchester 6.3.5-200.fc38.x86_64
Fedora 38
Intel xeon D1521
32GB DDR4 ECC RAM
2x 500GB SSDs
no gpu

**Describe the bug**
Logs get spammed with following error when opening webui
![image](https://github.com/go-skynet/LocalAI/assets/83671398/11cf2560-f908-4579-ab7f-fb287a78cdec)

**To Reproduce**
Follow this [example](https://github.com/go-skynet/LocalAI/tree/master/examples/chatbot-ui-manual)

**Expected behavior**
Run without any problems

**Logs**

![image](https://github.com/go-skynet/LocalAI/assets/83671398/9be191d7-a89c-4269-909e-69cdf40ed344)

**Additional context**

**Comment by sebastien-prudhomme** [comment]
Did you modify the docker-compose file? Connection port is 8080 not 8199 as displayed in your logs

**Comment by DenuxPlays** [comment]
> Did you modify the docker-compose file? Connection port is 8080 not 8199 as displayed in your logs

yeah I changed all ports from 8080 to 8199 because 8080 is already in use and can not be freed.
I asked [here](https://github.com/go-skynet/LocalAI/discussions/484) on how to do it correctly or if this is even possible but got no response.

**Comment by sebastien-prudhomme** [comment]
You need to add an env variable in your docker-compose file so that LocalAI listens on your port: "ADDRESS=:8199"

**Comment by DenuxPlays** [comment]
> You need to add an env variable in your docker-compose file so that LocalAI listens on your port: "ADDRESS=:8199"

Can we document this please?
